invenio_rdm_records/services/config.py

```diff
@@ -167,3 +169,13 @@ class RDMFileDraftServiceConfig(FileServiceConfig):
 
     record_cls = RDMDraft
     permission_policy_cls = RDMRecordPermissionPolicy
+
+    file_links_list = {
```

Comment:
was all this missing? was any functionality broken?

Reply:
I guess it was work from other sprints (QA?) to improve/fix the links?